directives/submit.py

```diff
@@ -101,6 +102,7 @@ def run(self):
             u'min_group_size': data.get('min_group_size', env.config.default_min_group_size),
             u'max_group_size': data.get('max_group_size', env.config.default_max_group_size),
             u'points_to_pass': self.options.get('points-to-pass', data.get('points_to_pass', 0)),
+            u'status': self.options.get('status', data.get('status','unlisted'))
```

Comment:
Move this out of the `data.update` step and do something like: `data.setdefault('status', self.options.get('status', 'unlisted'))`